todo/src/domain/todo.rs

```diff
@@ -1,26 +1,30 @@
 use std::fmt;
 
+use uuid::Uuid;
+
 #[derive(Debug, Clone, PartialEq)]
 pub struct Todo {
     pub message: String,
     pub done: bool,
+    pub id: Option<Uuid>,
 }
 
 impl Todo {
     pub fn new(message: String) -> Self {
         Todo {
             message,
             done: false,
+            id: None,
```

Comment:
You should generate the Uuid here.

Reply:
Otherwise, one would introduce oneself in an unwrap hell, as observed below. t-t